### DISCUSSION TITLE
Booksearch Example not working anymore in Expo-go and snack.expo.io as well

### DISCUSSION THREAD

**Discussion by dipenparmar12** [appbaseio/reactivesearch#2131] [General]
Booksearch Example not working anymore in Expo-go and snack.expo.io as well
[`booksearch example app with Snack Editor`]( https://snack.expo.io/@metagrover/booksearch)

**Comment by SavvyShah** [comment]
Thanks for reporting the issue. If you want a working example then you can try this example at [`/packages/native-examples`](https://github.com/SavvyShah/reactivesearch/tree/expo-example/packages/native-examples).
You can follow this steps.
1. Clone the forked repo `git clone https://github.com/SavvyShah/reactivesearch.git`
2. Checkout to the `git checkout fix/change-xdate-to-dayjs`
3. Checkout to the commit `git checkout fcfcee32baaf4427e75b4f4a9de0931e35b44b16`
4. Install deps and start the compiler from root `yarn && yarn dev:web`
5. Install deps and run the example from packages/native-examples/all-in-one `yarn && yarn start`
6. Install expo on mobile and scan the QR code.
